pandas/tools/tests/test_util.py

```diff
@@ -401,6 +401,32 @@ def test_downcast(self):
             res = pd.to_numeric(data, downcast=downcast)
             tm.assert_numpy_array_equal(res, expected)
 
+        # check that the smallest and largest values in each integer type pass to each type.
+        int8 = [-128, 127]
```

Comment:
don't do this explicty, use ``np.iinfo(dtype).min/max`` as these vary by platform